bindings/java/src/test/com/apple/foundationdb/test/AsyncStackTester.java

```diff
@@ -483,6 +483,14 @@ else if (op == StackOperation.TENANT_DELETE) {
 				inst.push(TenantManagement.deleteTenant(inst.context.db, tenantName));
 			}, FDB.DEFAULT_EXECUTOR);
 		}
+		else if (op == StackOperation.TENANT_LIST) {
+			return inst.popParams(3).thenAcceptAsync(params -> {
+				byte[] begin = (byte[])params.get(0);
+				byte[] end = (byte[])params.get(1);
+				int limit = StackUtils.getInt(params.get(2));
+				inst.push(TenantManagement.listTenants(inst.context.db, begin, end, limit));
+			}, FDB.DEFAULT_EXECUTOR);
+		}
```

Comment:
Something like this is the right idea. There are a couple other things to do, though, to make this work:

1. Update bindings/bindingtester/spec/tenantTester.md to describe the behavior of the new list operation
2. Update bindings/bindingtester/tests/api.py to generate tests with the new list operation
3. Add the list operation to the Python binding tester in bindings/python/tests/tester.py
4. Run the bindingtester in Joshua to make sure that the bindings agree in their implementations

Reply:
Initial changes made but not tested yet. Will need to iterate over a few parts